components/pkg-export-docker/src/build.rs

```diff
@@ -99,12 +99,12 @@ impl<'a> BuildSpec<'a> {
             channel: m
                 .value_of("CHANNEL")
                 .map(ChannelIdent::from)
-                .unwrap_or_default(),
+                .unwrap_or_default(), // or configured_value?
             base_pkgs_url: m.value_of("BASE_PKGS_BLDR_URL").unwrap_or(&default_url),
             base_pkgs_channel: m
                 .value_of("BASE_PKGS_CHANNEL")
                 .map(ChannelIdent::from)
-                .unwrap_or_default(),
+                .unwrap_or_default(), // or configured_value?
```

Comment:
Since we've only got one environment variable for channels, and this base packages channel is a rather esoteric use case, we should continue using `default` here.

If we're going to honor `HAB_BLDR_CHANNEL` for the exporter, I think it makes more sense to use it for the channel you're grabbing your main package from.